main.go

```diff
@@ -17,10 +17,11 @@ import (
 func init() {
 	log.SetFormatter(&log.JSONFormatter{})
 }
-
+//TODO set Prefix by echo Router library?
 func main() {
 	app := cli.NewApp()
 	app.Author = "MEDIGO GmbH"
+	os.Setenv("PREFIX_ROUTE", "/laika") //Added a prefix as an environment variable
```

Comment:
>>>but anyway the default prefix should be no prefix, so it doesn't break our current deploy
What do you mean with this line? 
I set a new pull request, cleaner and nicer  😄